redis/conn.go

```diff
@@ -213,6 +232,15 @@ func DialURL(rawurl string, options ...DialOption) (Conn, error) {
 		return nil, fmt.Errorf("invalid database: %s", u.Path[1:])
 	}
 
+	if u.Scheme == "rediss" {
+		// insert a default DlialTLS at position 0, so we don't override any
+		// user provided *tls.Config elsewhere in options
+		t := DialTLS(&tls.Config{InsecureSkipVerify: true})
```

Comment:
I suspect it's the sanest option as most setups I've seen in the wild currently make fake certs with the intent to just encrypt the connection (vs. also ensuring you are connecting to what you think you are).

Users who need more security can still `DialURL(url, DialTLS(&tls.Config{<better security options>})`.